.circleci/cimodel/data/caffe2_build_data.py

```diff
@@ -27,15 +28,24 @@ def get_children(self):
         return [self.child_constructor()(self, k, v) for (k, v) in self.subtree]
 
     def is_build_only(self):
-        if str(self.find_prop("language_version")) == "onnx_py3.6":
+        if (str(self.find_prop("language_version")) == "onnx_py3.6_part1") or \
```

Comment:
Since it's already wrapped with (), no need for backslash

Reply:
I'm wrapping each part separately, so I think backslach would be needed. However, I'll remove the wrapping since it's not needed.